modules/ding_nodelist/ding_nodelist.module

```diff
@@ -1252,6 +1253,10 @@ function _ding_nodelist_sort_nodes(array $nodes, array $conf) {
         $sorted_nodes[][$date] = $node;
       }
       elseif ($conf['sort_field'] == DING_NODELIST_SORT_BY_CREATION_DATE) {
+        if (array_key_exists($node->created, $sorted_nodes)) {
+          $index = $i++ ?? (int) 0;
+          $node->created = $node->created . '_' . $index;
```

Comment:
Think that this can be achieved without altering node's data.

Reply:
Agree, it wasn't the best idea. Fixed.